website/docs/advanced/override_grammar/basic.md

```diff
@@ -147,6 +147,29 @@ Quoted strings can accept any value between the quotes, but some characters need
 </div>
 </div>
 
+It may be necessary to use multiple pairs of quotes to prevent your
+shell from consuming quotation marks before they are passed to hydra.
+
+```shell
+$ python my_app.py '+foo="{a: 10}"'
+foo: '{a: 10}'
+
+$ python my_app.py '+foo={a: 10}'
+foo:
+  a: 10
+
+```
+
+Here are some best practices around quoting in CLI overrides:
+- Quote the whole key=value pair with single quotes, as in the first two
+  examples above. These quotes are for the benefit of the shell.
+- Do not quote keys.
+- Only quote values if they contain a space. It will work if you always quote
+  values, but it will turn numbers/dicts/lists into strings (as in the first
+  example above).
+- When you are quoting values, use double quotes to avoid collision with the
+  outer single quoted consumed by the shell.
+
```

Comment:
These are good basic rules that should cater for almost all cases.

BTW double quoting in the shell is more general than single quoting, as you can specify any character through the use of escapes (you can't have literal single quotes within single quoted args for example). But I wouldn't bother mentioning that in the doc as it's shell specific